src/AppBundle/Controller/WebhookController.php

```diff
@@ -24,8 +25,12 @@ public function githubAction(Request $request)
         if ($data === null) {
             return new JsonResponse(['error' => 'Invalid JSON body'], 500);
         }
+        try {
+            $event = $this->get('app.webhook_resolver')->resolve($data);
```

Comment:
this fixes error 500 triggered by PrestonBot when the event is not recognized by Lp digital Github parser.